chain/network/src/peer.rs

```diff
@@ -731,6 +742,18 @@ impl StreamHandler<Result<Vec<u8>, ReasonForBan>> for Peer {
                 return;
             }
         };
+        // Drop duplicated messages routed within 1s
+        if let PeerMessage::Routed(msg) = &peer_msg {
+            let key = (msg.author.clone(), msg.target.clone(), msg.signature.clone());
+            let now = Utc::now();
+            if let Some(time) = self.routed_message_cache.cache_get(&key) {
+                if time.signed_duration_since(now).num_seconds() >= 1 {
```

Comment:
should this be `<=` instead of `>=`? Also we should make this number a constant

Reply:
fixed